pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -813,6 +851,7 @@ type BackupSpec struct {
 	Resources                corev1.ResourceRequirements  `json:"resources,omitempty"`
 	RuntimeClassName         *string                      `json:"runtimeClassName,omitempty"`
 	PITR                     PITRSpec                     `json:"pitr,omitempty"`
+	Configuration            BackupConfig                 `json:"configuration,omitempty"`
```

Comment:
@egegunes @hors wdyt about this structure, `Configuration` is `BackupConfig` that then holds `BackupOptions` and `RestoreOptions`? I'm like not that happy about it honestly, feels like we could do better, but nothing comes to me currently.

Reply:
@inelpandzic please ask @spron-in about it. All changes related to cr should be discussed with Sergey.